ex02.ex

```diff
@@ -53,7 +55,8 @@ defmodule Ex02 do
   # if the first two elements of a list are equal                #
   ################################################################
 
-  first2equal = your_anonymous_function([])
+  # first2equal = your_anonymous_function([])
+  first2equal = fn [ a, b | tail ] -> a === b end
```

Comment:
No deductions, but I'm interested to see you used `===` rather than `==`. This hadn't occurred to me before.